js/windowMgr.js

```diff
@@ -73,6 +73,7 @@ function doCreateMainWindow(initialUrl, initialBounds) {
         show: true,
         minWidth: MIN_WIDTH,
         minHeight: MIN_HEIGHT,
+        titleBarStyle: 'hidden',
```

Comment:
also looks like we loose capability to move window around?  maybe change in client fixes this???